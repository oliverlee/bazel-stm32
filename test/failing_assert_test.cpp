#include <cassert>

auto main() -> int
{
  const auto x = 42;
  assert(41 == x);
}
