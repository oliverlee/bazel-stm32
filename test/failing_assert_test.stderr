assertion "41 == x" failed: file "test/failing_assert_test.cpp", line 6, function: int main()
